contrib/gofiber/fiber.v2/option.go

```diff
@@ -8,6 +8,8 @@ package fiber
 import (
 	"math"
 
+	"github.com/gofiber/fiber/v2"
```

Comment:
This import group (external) goes after the local ones.

Reply:
Oh, that was news to me. Just fixed it as the lint step in the Circle CI action suggested.

Reply:
I added a commit fixing the package as I could see this was wrong in multiple places.